blocks/library/embed/index.js

```diff
@@ -114,6 +121,10 @@ function getEmbedBlockSettings( { title, icon, category = 'embed', transforms, k
 							return;
 						}
 						response.json().then( ( obj ) => {
+							if ( obj.html && -1 !== obj.html.indexOf( 'class="wp-embedded-content"' ) ) {
```

Comment:
Minor: Complementing the above remark, `_.includes` also automatically handles falsey values, so this could be simplified to:

```js
if ( includes( obj.html, 'class="wp-embedded-content"' ) ) {
```

Reply:
Nice, i will include this for sure. Thanks.